pkg/build/controller/strategy/docker_test.go

```diff
@@ -73,7 +71,9 @@ func TestDockerCreateBuildPod(t *testing.T) {
 	if *actual.Spec.ActiveDeadlineSeconds != 60 {
 		t.Errorf("Expected ActiveDeadlineSeconds 60, got %d", *actual.Spec.ActiveDeadlineSeconds)
 	}
-	for i, expected := range []string{buildutil.BuildWorkDirMount, dockerSocketPath, crio.CrioSocket, DockerPushSecretMountPath, DockerPullSecretMountPath} {
+	//for i, expected := range []string{buildutil.BuildWorkDirMount, dockerSocketPath, crio.CrioSocket, DockerPushSecretMountPath, DockerPullSecretMountPath} {
+	// TODO cadvisor and crio disagree.  One says '/var/run/crio/crio.sock', the other says "/var/run/crio.sock"
```

Comment:
need @mrunalp to resolve what this should be.

Reply:
If it has changed, that's going to break builds on crio runtimes due to: https://github.com/openshift/origin/blob/master/pkg/build/controller/strategy/util.go#L26.  I don't remember why that is not using a constant like the test is. It probably should be.

Reply:
It should be /var/run/crio/crio.sock for 1.9
cc: @runcom Can you make sure our system container image has the right setting for crio.conf

Reply:
should be `/var/run/crio/crio.sock` from 1.8 afaict

Reply:
we've only ever mounted /var/run/crio.sock (https://github.com/openshift/origin/blob/master/pkg/build/controller/strategy/util.go#L26) so this is broken on 1.8 then.